tests/test_decode_tokens.py

```diff
@@ -286,3 +294,10 @@ def test_malformed_token(app):
     with pytest.raises(DecodeError):
         with app.test_request_context():
             decode_token(invalid_token)
+
+
+def test_jwt_headers(app, default_access_token, default_access_token_header):
+    token = encode_token(app, default_access_token, headers=default_access_token_header)
```

Comment:
Because your test is using `encode_token`, which is a unit test wrapper for `jwt.encode`, this isn't actually testing the code path that would be used in flask_jwt_extended, it's acting more of a unit test for the underlying pyjwt library. Could we change this test to use `create_access_token()` and `create_refresh_token()` instead?

Reply:
Makes sense, we can change this test through `create_access_token()` and `create_refresh_token()` funcs.